tests/unit/test_kaniko.py

```diff
@@ -0,0 +1,18 @@
+from gcip import Pipeline, PredefinedVariables
+from tests import conftest
+from gcip.addons.kaniko import jobs as kaniko
+
+
+def test_default_kaniko_job(gitlab_ci_environment_variables):
+    pipeline = Pipeline()
+
+    pipeline.add_children(
+        kaniko.execute(
+            image_name="thomass/gcip",
+            enable_push=(PredefinedVariables.CI_COMMIT_TAG or PredefinedVariables.CI_COMMIT_BRANCH == "main"),
+            dockerhub_user_env_var="DOCKER_USER",
```

Comment:
Just in general, why do you use *_env_var arguments. 
I think we could hardcode this iside the job and write a short documentation about, how the variable should be named. That would simplify the function call. 
I think this is because of the question: What happens if the `DOCKER_USER` is used elsewhere in a different context. 
If so, I think we shouldn't care about this, cause we should just give simple interfaces.

Reply:
Maybe we can instead use `registry_user_env_var` and so forth.